knowledge-base/live-traffic-pane.md

```diff
@@ -497,6 +513,17 @@ All options will display a **Save** prompt window. To save the desired sessions:
 Alternatively, you can save all captured traffic through the **Save** button from the toolbar.
 
 
+### Exporting
```

Comment:
We will only work with Selected sessions now, there are no dropdowns with selected/unselected/all sessions